recipes/hiveplot/meta.yaml

```diff
@@ -0,0 +1,38 @@
+{% set version = "0.1.8.4" %}
+{% set pkgname = "hiveplot" %}
+
+package:
+    name: {{ pkgname }}
+    version: {{ version }}
+
+source:
+    fn: {{ pkgname }}-{{ version }}.tar.gz
+    url: https://pypi.io/packages/source/{{ pkgname[0] }}/{{ pkgname }}／{{ pkgname }}-{{ version }}.tar.gz
```

Comment:
That '/' after the first `{{ pkgname }}` is not a slash! It's a unicode fullwidth solidus apparently...